src/register/data/utils.js

```diff
@@ -75,8 +75,7 @@ export const isFormValid = (
 
   if (getConfig().SHOW_CONFIGURABLE_EDX_FIELDS) {
     if (!configurableFormFields?.country?.displayValue) {
-      fieldErrors.country = formatMessage(messages['empty.country.field.error']);
```

Comment:
I would remove the message if it is not used anymore

Reply:
But I'm not sure (more diff with upstream)

Reply:
With these changes, we should wait for a meeting with the client, and there ask him about the Country field (should we leave it).

Reply:
this is deleted